app/Models/User.php

```diff
@@ -1023,7 +1023,10 @@ public function isAdmin()
 
     public function isChatAnnouncer()
     {
-        return $this->findUserGroup(app('groups')->byIdentifier('announce'), true) !== null;
+        $token = $this->token();
+        return $token !== null && !$token->delegatesOwner() && !$token->isOwnToken()
+            ? false
+            : $this->findUserGroup(app('groups')->byIdentifier('announce'), true) !== null;
```

Comment:
This change may cause problems if people have been using lazer to send additional messages to announcement channels